python/django/python-hello-world/src/helloapp/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
```

Comment:
nit: remove empty line

Reply:
https://www.python.org/dev/peps/pep-0008/#blank-lines.   Class definitions should be preceded by 2 blank lines.